.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
+
+[*]
+indent_size = 2
+continuation_indent_size = 4
```

Comment:
Am I correct it's for instance for fluent API when you put invocations on new lines?

https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties
https://github.com/editorconfig/editorconfig-jetbrains/issues/31